adapters/oidc/spring-boot/src/main/java/org/keycloak/adapters/springboot/KeycloakSpringBootConfiguration.java

```diff
@@ -209,7 +209,8 @@ public void customize(Server server) {
                 }
             }
 
-            WebAppContext webAppContext = server.getBean(WebAppContext.class);
+            WebAppContext webAppContext = (WebAppContext) server.getHandler();;
```

Comment:
I'm not sure about the `getHandler()` call, as Jetty allows for deploying multiple contexts in a contextcollection for which this might not work. I'm not familiar with Jetty in multi-context deployments though, so I might be overthinking this. I'm sure your fix will be an improvement nonetheless...

<sub>p.s. I'm not meaning to be nitpicky :blush:, but both a double semicolon and an unnecessary empty line leave something to be desired...</sub>